src/OpenTelemetry/Trace/ActivitySourceAdapter.cs

```diff
@@ -101,8 +102,9 @@ private void RunGetRequestedData(Activity activity)
                 activity.Links);
 
             var samplingDecision = this.sampler.ShouldSample(samplingParameters);
```

Comment:
This seems to be a misleading variable name introduced before this PR. I guess it should be `samplingResult` instead of `samplingDecision`.